src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final byte dateIndex = 0;
+    static final byte nameIndex = 1;
+    static final byte workingDaysIndex = 2;
+    static final byte hourlyRateIndex = 3;
+    static final DateTimeFormatter formatter =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder report = new StringBuilder("Report for period " + dateFrom
+                + " - " + dateTo);
+        LocalDate dtFrom = LocalDate.parse(dateFrom, formatter).minusDays(1);
+        LocalDate dtTo = LocalDate.parse(dateTo, formatter).plusDays(1);
```

Comment:
```suggestion
        LocalDate localDateFrom = LocalDate.parse(dateFrom, formatter);
        LocalDate localDateTo = LocalDate.parse(dateTo, formatter);
```
instead of minus and plus 1 day you can check if the date that you get from the array equals dateFrom or dateTo